lib/telemetry_metrics_statsd.ex

```diff
@@ -155,10 +155,12 @@ defmodule TelemetryMetricsStatsd do
           | {:metrics, [Metrics.t()]}
           | {:mtu, non_neg_integer()}
           | {:prefix, String.t()}
+          | {:formatter, TelemetryMetricsStatsd.Formatter}
```

Comment:
Is this valid typespec? I don't use them outside of library code and not sure how to reference "type of behaviour"

Reply:
This typespec means that only this module is a valid option. There are no really types for behaviours, so I would just go for a `module()` here.